certbot-apache/certbot_apache/obj.py

```diff
@@ -167,6 +167,19 @@ def __str__(self):
                 active="Yes" if self.enabled else "No",
                 modmacro="Yes" if self.modmacro else "No"))
 
+    def display_repr(self):
+        """Return a representation of VHost to be used in dialog"""
+        return (
+            "File: {filename}\n"
+            "Addresses: {addrs}\n"
+            "Names: {names}\n"
+            "HTTPS: {https}\n\n".format(
```

Comment:
nit: The trailing empty line on the last entry looks a little awkward to me:
```
-------------------------------------------------------------------------------
1: File: /etc/apache2/sites-enabled/bmatch.conf
Addresses: *:80
Names: geqgeqoc.example.org
HTTPS: No

2: File: /etc/apache2/sites-enabled/amatch.conf
Addresses: *:80
Names: a.example.org
HTTPS: No

3: File: /etc/apache2/sites-enabled/wildcard.conf
Addresses: *:80
Names: abc.def.example.org, *.example.org
HTTPS: No

-------------------------------------------------------------------------------
```
Unless you prefer it that way, can we remove it?

Reply:
It looks a lot cleaner without the extra newline, I agree. Even though handling that requires a few more lines in the code. This is now implemented.